frontend/badges/100solvedProblems/query.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+    DISTINCT `u`.`user_id`
+FROM
+    `Problems` AS `p`
+INNER JOIN
+    `Submissions` AS `s` ON `p`.`problem_id` = `s`.`problem_id`
+INNER JOIN
+    `Runs` AS `r` ON `r`.`run_id` = `s`.`current_run_id`
+INNER JOIN
+    `Identities` AS `i` ON `s`.`identity_id` = `i`.`identity_id`
+INNER JOIN
+    `Users` AS `u` ON `u`.`main_identity_id` = `i`.`identity_id`
+WHERE
+    `r`.`verdict` = "AC" AND `s`.`type` = "normal"
+GROUP BY
+    `u`.`user_id`
+HAVING
+    COUNT(*) >= 100;
```

Comment:
```suggestion
    COUNT(DISTINCT p.problrm_id) >= 100;
```

esta sintaxis es válida? si no, necesitamos hacer algo equivalente. puedes cambiar la prueba de tal manera que un usuario tenga 100 AC en un solo problema y aseverar que no reciba el badge?

Reply:
Sí es válida,
Agregaré una prueba más que haga lo que dices.